test/iamvery/phoenix/live_view/test_helpers_test.exs

```diff
@@ -5,19 +5,22 @@ end
 
 defmodule Phoenix.LiveViewTest do
   @moduledoc "A test fake to stand in for the real Phoenix module"
-  @html "<html>Home. Edit Link. Link updated successfully. It can't be blank</html>"
+  @html "<html>Home. <div>Edit Link</div>. Link updated successfully. It can't be blank</html>"
+  @element "<div>Edit Link</div> SPECIAL"
   def live(_, _), do: {:ok, :live, @html}
   def form(:live, _, _), do: :form
-  def element(:live, _), do: :live
-  def element(:live, _, _), do: :live
+  def element(:live, _), do: :element
+  def element(:live, _, _), do: :element
   def has_element?(:live, _, "yes"), do: true
   def has_element?(:live, _, "no"), do: false
   def has_element?(:live, ".no", _), do: false
   def has_element?(:live, _, nil), do: true
   def follow_redirect(@html, :conn), do: {:ok, :live, @html}
   def follow_redirect(redirect, :conn), do: redirect
   def render(:live), do: @html
+  def render(:element), do: @element
   def render_click(:live), do: @html
+  def render_click(:element), do: @html
   def render_change(:form), do: @html
   def render_submit(:form), do: @html
   def assert_patch(:live, _), do: {:ok, :live, @html}
```

Comment:
🤔 All this crazy stubbing is getting difficult to reason about. Should probably consider using the real helpers. Not sure how best to set that up.

Reply:
Fixed. See #21